ballerina/errors.bal

```diff
@@ -14,10 +14,18 @@
 // specific language governing permissions and limitations
 // under the License.
 
+# Defines the generic error type for the `persist` module.
 public type Error distinct error;
 
+# Represents an error that occurs when an attempt is made to manipulate a field which does not exist in an entity.
 public type FieldDoesNotExist distinct Error;
+
+# Represents an error that occurs when an attempt is made to insert a value into a record which cannot be directly
+# inserted into.
 public type InvalidInsertion distinct Error;
+
+# Represents an error that occurs when an attempt is to perform an operation which violates a foreign key constraint.
 public type ForeignKeyConstraintViolation distinct Error;
 
+# Represents an error that occurs when an attempt is made to retrieve a record using a key which does not exist.
```

Comment:
```suggestion
# Represents an error that occurs when an attempt is made to retrieve a record using a key, which does not exist.
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)